### PR TITLE
Skip extremely long chebyshev/legendre tests introduced in #78304

### PR DESCRIPTION
**Pull request by janeyx99**
See https://github.com/pytorch/pytorch/issues/79528

### CODE REVIEW

**Review by mruberry — APPROVED** [review]
Stamped

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -19610,6 +19610,7 @@ def error_inputs_mean(op_info, device, **kwargs):
         dtypes=all_types_and(torch.bool),
         promotes_int_to_float=True,
         skips=(
+            DecorateInfo(unittest.skip("Skipping - testing takes an unreasonably long time, #79528")),
```
Comment (janeyx99) [inline torch/testing/_internal/common_methods_invocations.py:19613]: I'm not sure if I'm using the DecorateInfos correctly here--is this how I skip the op for all test classes? Or would it be more like
```suggestion
            DecorateInfo(unittest.skip("Skipping - testing takes an unreasonably long time, #79528"), 'TestCommon'),
```

**Review by malfet — APPROVED** [review]
LGTM, can you please mention how long does it take to finish it?

**Comment by janeyx99** [comment]
> LGTM, can you please mention how long does it take to finish it?

Total of ~8.5hrs compared to the current almost 10.5-11hrs on trunk

**Pull request closed without merging**